libraries/AdaptiveExpressions/Options.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AdaptiveExpressions
+{
+    /// <summary>
+    /// Options used to define evaluation behaviors.
+    /// </summary>
+    public class Options
+    {
+        public delegate object GetValueDelegate(string path);
```

Comment:
>GetValueDelegate [](start = 31, length = 16)

I don't think this is needed, but if it is you need to document it.

Reply:
yes, replace by Func<string ,object>

Reply:
I though you were going to use Func<string, object>?  

---
In reply to: [400615412](https://github.com/microsoft/botbuilder-dotnet/pull/3635#discussion_r400615412) [](ancestors = 400615412)